docs/docsite/rst/playbook_guide/playbooks_variables.rst

```diff
@@ -360,7 +360,7 @@ Understanding variable precedence
 Ansible does apply variable precedence, and you might have a use for it. Here is the order of precedence from least to greatest (the last listed variables override all other variables):
 
   #. command line values (for example, ``-u my_user``, these are not variables)
-  #. role defaults (defined in role/defaults/main.yml) [1]_
+  #. role defaults (defined in role/defaults/main.yml (also main.yaml or main)) [1]_
```

Comment:
i would just skip directory and add ref to role structure page

Reply:
Same as above. I think that showing the path and file name right away can be useful. I would prefer just adding the ref to the end

Reply:
It's actually more complex that `main.yml, main.yaml or main`` - given it has nested directories etc, it's not possible to put an accurate short definition here. So I'd say go with @bcoca suggestion and add a link to the role structure section instead. Something like
#. role defaults (as defined in :ref:`role_structure`)[1]_